Unhandled go type LONGCHAR

# Expected behaviour
```
When running make examples, following is expected:
Opening database
Creating table 'simple'
Writing a=2147483647, b='a string' to table
Querying values from table
Displaying results of query
| a          | b                              |
| 2147483647 | a string             
```
# Current behaviour
Currently the following issue has been seen
```
Opening database
Creating table 'simple'
Writing a=2147483647, b='a string' to table
Querying values from table
Displaying results of query
| a          | b                              |
2020/05/25 09:46:34 Failed: Error reading rows: Unhandled Go type: LONGCHAR
exit status 1
make: *** [Makefile:24: examples/cgo/simple] Error 1
```
# Steps to reproduce
Run 'make examples' with any recent PL of ASE 16.0 SP02 or 16.0 SP03

# Environment
OS: Linux

## Client

OS: Linux 
go version: go version go1.14 linux/amd64
go-ase version / commit hash: 

## Server

OS: RHEL 6.10
ASE version: 16.0 SP02 PL09/16.0 SP03 PL08

--- a/cgo/rows.go
+++ b/cgo/rows.go
@@ -282,7 +282,7 @@ func (rows *Rows) Next(dest []driver.Value) error {
 			t = t.Add(time.Duration(dur) * time.Microsecond)
 
 			dest[i] = t
-		case types.CHAR, types.VARCHAR, types.TEXT:
+		case types.CHAR, types.VARCHAR, types.TEXT, types.LONGCHAR:
 			dest[i] = C.GoString((*C.char)(rows.colData[i]))
 		case types.BINARY, types.IMAGE:
 			dest[i] = C.GoBytes(rows.colData[i], rows.dataFmts[i].maxlength)

--- a/cgo/statement.go
+++ b/cgo/statement.go
@@ -302,7 +302,7 @@ func (stmt *statement) exec(args []driver.NamedValue) error {
 
 			ptr = C.CBytes(bs)
 			defer C.free(ptr)
-		case types.CHAR:
+		case types.CHAR, types.LONGCHAR:
 			ptr = unsafe.Pointer(C.CString(arg.Value.(string)))
 			defer C.free(ptr)
 

--- a/cgo/statement.go
+++ b/cgo/statement.go
@@ -302,14 +302,14 @@ func (stmt *statement) exec(args []driver.NamedValue) error {
 
 			ptr = C.CBytes(bs)
 			defer C.free(ptr)
-		case types.CHAR, types.LONGCHAR:
+		case types.CHAR:
 			ptr = unsafe.Pointer(C.CString(arg.Value.(string)))
 			defer C.free(ptr)
 
 			datalen = len(arg.Value.(string))
 			datafmt.format = C.CS_FMT_NULLTERM
 			datafmt.maxlength = C.CS_MAX_CHAR
-		case types.TEXT:
+		case types.TEXT, types.LONGCHAR:
 			ptr = unsafe.Pointer(C.CString(arg.Value.(string)))
 			defer C.free(ptr)
 

